Makefile

```diff
@@ -45,6 +45,7 @@ endif
 CGO_CFLAGS := -I$(FFMPEG_BUILD)/include
 GOFLAGS := -buildvcs=false
 export PKG_CONFIG_PATH=$(FFMPEG_BUILD)/lib/pkgconfig
+export PATH := $(PATH):$(shell go env GOPATH)/bin
```

Comment:
What are we using `PATH` for here?

Reply:
Just so that bins in go path are available (for `artifact` calls).